website/blog/2024-05-17-whats-next.md

```diff
@@ -0,0 +1,68 @@
+---
+slug: whats-next
+title: What's next for MACI?
+description: Dive into the next MACI protocol changes and improvements.
+authors:
+  name: ctrlc03
+  title: MACI dev
+  url: https://x.com/ctrlc03
+  image_url: https://avatars.githubusercontent.com/u/93448202?v=4
+tags: [voting, security, anonymity]
```

Comment:
```suggestion
tags: [voting, security, anonymity, roadmap, grants]
```
I've been using the `roadmap` tag: https://maci.pse.dev/blog/tags/roadmap

And perhaps makes sense to have one for `grants` moving forward?